crates/infra/cli/src/toolchains/napi/compiler.rs

```diff
@@ -125,11 +130,13 @@ fn compile_root_package(node_binary: Option<&Path>) -> Result<()> {
         let source = package_dir.join(file_name);
         let destination = output_dir.join(file_name);
 
-        std::fs::copy(source, destination)?;
+        if source.exists() {
```

Comment:
I'm a bit worried about this condition, as it allowing shipping a release without a license or other critical files (which can accidentally happen as we move stuff around). I think we should fail the build if not found.

To avoid further adding to this module logic, what do you think of adding dummy/empty versions of these files under `testlang`?